spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -4233,6 +4234,27 @@ def __in_block_comment(cursor):
         cursor3.endEditBlock()
         return True
 
+    def create_new_cell(self):
+        firstline = '#%% ' + self.get_line_separator()
```

Comment:
Following comment at https://github.com/spyder-ide/ux-improvements/issues/30#issuecomment-914371091 and to prevent a trailing white space warning this could be

```suggestion
        firstline = '# %%' + self.get_line_separator()
```

What do you think @ccordoba12 ?

Reply:
Agreed, thanks for spotting that @dalthviz!